src/components/admin/Register.js

```diff
@@ -0,0 +1,135 @@
+/* eslint-disable no-unused-vars */
+/** @jsxImportSource @emotion/react */
+import "twin.macro";
+import axios from "axios";
+import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { logout } from "../../utils/admin";
+import { useUserState } from "../../utils/userState";
+import { GenericInput } from "../form/GenericInput";
+import { IconLogoBlue, WordmarkLogo } from "../icons/Icons";
+
+export const Register = () => {
+  const navigate = useNavigate();
+  const [, setUser] = useUserState();
+
+  const [email, setEmail] = useState("");
+  const [pass, setPass] = useState("");
+  const [confirm, setConfirm] = useState("");
+  const [error, setError] = useState(null);
+  const [loading, setLoading] = useState(false);
+
+  const handleError = (errorMessage) => {
+    setError(errorMessage);
+    setTimeout(() => {
+      setError(null);
+    }, 3000);
+  };
+
+  /*
+  REPLACE WITH REGISTRATION LOGIC
```

Comment:
The endpoint already exists and has been public. I'm pretty sure its just auth/signup afaik. Shouldn't be too hard to change. i can send body of what we're looking for then. 

Things to consider: This needs to have a successful loading state / error handling on submission. I think the PR should also have the signup request work successfully. 

This also should not be merged / visible and available until the separation of the moderation tab is created. Can talk about more if needed. Good to have the code ready to go though.

Reply:
Yeah was def not planning on merging now, but just a start for when we are ready for it